packages/components/src/Metadata.tsx

```diff
@@ -1,55 +1,173 @@
+                        fontSize: 20,
+                    }}
+                >
+                    {title}
+                </p>
+            )}
+
+            {description && (
+                <p
+                    sx={{
+                        fontSize: 16,
+                        opacity: 0.8,
+                        my: 0,
+                    }}
+                >
+                    {description}
+                </p>
+            )}
+            <Flex sx={{ justifyContent: 'space-between', flexWrap: 'wrap' }}>
+                {creator && (
+                    <Box mt={3}>
+                        <p
+                            sx={{
+                                fontSize: 14,
+                                fontWeight: 500,
+                                my: 0,
+                            }}
+                        >
+                            Created by
+                        </p>
+                        <a
+                            href={`https://explorer.testnet.near.org/accounts/${creator}`}
+                            target="_blank"
+                            sx={{
+                                fontSize: 16,
+                                fontWeight: 600,
+                                my: 0,
+                                color: 'inherit',
+                                textDecoration: 'none',
+                                '&:hover': {
+                                    opacity: 0.8,
+                                },
+                            }}
+                        >
+                            {creator}
+                        </a>
+                    </Box>
+                )}
+
+                {owner && (
+                    <Box mt={3}>
+                        <p
+                            sx={{
+                                fontSize: 14,
+                                fontWeight: 500,
+                                my: 0,
+                            }}
+                        >
+                            Owned by
+                        </p>
+                        <a
+                            href={`https://explorer.testnet.near.org/accounts/${owner}`}
+                            target="_blank"
+                            sx={{
+                                fontSize: 16,
+                                fontWeight: 600,
+                                my: 0,
+                                color: 'inherit',
+                                textDecoration: 'none',
+                                '&:hover': {
+                                    opacity: 0.8,
+                                },
+                            }}
+                        >
+                            {owner}
+                        </a>
+                    </Box>
+                )}
+            </Flex>
+        </>
+    )
+}
+function MetadataLoading() {
+    return (
+        <>
+            <Placeholder width="40%" style={{ mt: 3 }} />
```

Comment:
Hey @achrafismyname, did you make sure all these percentage widths work well on different screen sizes (mobile, tablet)?

Reply:
Yes, because its a percentage of the parent element which is responsive